lib/pagseguro/transaction_request/request_serializer.rb

```diff
@@ -143,6 +151,155 @@ def serialize_extra_params(extra_params)
       def to_amount(amount)
         "%.2f" % BigDecimal(amount.to_s).round(2).to_s("F") if amount
       end
+
+      def xml_builder
+        Nokogiri::XML::Builder.new(encoding: PagSeguro.encoding) do |xml|
+          xml.send(:payment) {
+            xml.mode transaction_request.payment_mode
+            xml.method_ transaction_request.payment_method
```

Comment:
ahh ok... I really image these name is to discern the Nokogiri **method** and the **method** of the Object class.